apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -142,14 +144,19 @@ function SectionProgressSelector({
   };
 
   return (
-    <div className={styles.pageContent}>
+    <div
+      className={classNames(
+        styles.pageContent,
+        isInV2Navigaton && styles.navView
+      )}
+    >
       {displayV2 && (
         <ProgressBanners hasJustSwitchedToV2={hasJustToggledViews} />
       )}
       {toggleV1OrV2Link()}
 
       {displayV2 ? (
-        <SectionProgressV2 />
+        <SectionProgressV2 hideTopHeading={isInV2Navigaton} />
```

Comment:
There was a [discussion](https://codedotorg.slack.com/archives/C06ERUABG01/p1724781990022289) yesterday about whether we can remove the header from the v1 entirely. Hopefully we can just remove this logic 🤞

Reply:
Unfortunately we can't remove it from the v1 yet :(